packages/itwinui-css/src/dialog/dialog.scss

```diff
@@ -171,7 +171,7 @@ $iui-dialogue-resizable-min-height: 7.75rem; // 7.75rem = 124px = title bar heig
   padding: 0 var(--iui-size-m);
   overflow-y: auto;
   overflow-y: overlay;
-  max-height: calc(100vh - 6.75rem); // 6.75rem = 108px = button height + paddings + bottom bar
+  max-height: calc(100svh - 6.75rem); // 6.75rem = 108px = button height + paddings + bottom bar
```

Comment:
`svh` was added pretty recently to browsers: https://caniuse.com/viewport-unit-variants

so you could add a `vh` fallback like this:
```
max-height: calc(100vh - 6.75rem);
max-height: calc(100svh - 6.75rem);
```

in browsers that support `svh`, the last declaration of `max-height` will be used. in browsers that don't, the previous declaration will be used.

Reply:
Thanks for the info. I updated this.